charts/openshift-metering/templates/presto/hive-metastore-statefulset.yaml

```diff
@@ -164,3 +194,8 @@ spec:
         persistentVolumeClaim:
           claimName: {{ .Values.presto.spec.config.sharedVolume.claimName }}
 {{- end}}
+{{- if .Values.presto.spec.hive.metastore.config.tls.enabled }}
+      - name: hive-metastore-tls-secret
+        secret:
+          secretName: {{ .Values.presto.spec.config.tls.secretName }}
```

Comment:
this appears to be undefined, is it not supposed to be `secretName: "{{ .Values.presto.spec.hive.metastore.tls.secretName }}"`

Reply:
Actually it should be `"{{ .Values.presto.spec.hive.metastore.config.tls.secretName }}"` I believe. Note the `config`.